src/pl/senla/hotel/entity/services/HotelService.java

```diff
@@ -5,15 +5,15 @@
 
 public abstract class HotelService {
 
-    protected int idService = -1;
+    protected Integer idService;
     protected int idOrder;
-    protected String typeOfService;
+    protected TypeOfService typeOfService;
```

Comment:
just ServiceType

Reply:
what is not correct here? String was changed to Enum.

Reply:
minor naming issue, just corrected the class name